stacks/resoto/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add someengineering https://helm.some.engineering
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="resoto"
+CHART="someengineering/resoto"
+NAMESPACE="resoto"
+values=values.yaml
```

Comment:
You'll want this to be the Github hosted version of values.yaml, as below, both here and in the upgrade script:
```
if [ -z "${MP_KUBERNETES}" ]; then
  # use local version of values.yml
  ROOT_DIR=$(git rev-parse --show-toplevel)
  values="$ROOT_DIR/stacks/resoto/values.yml"
else
  # use github hosted master version of values.yml
  values="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/resoto/values.yml"
fi
```

Reply:
Thanks @mbeyt. I added the lines into deploy and upgrade.